markup/forms.md

```diff
@@ -0,0 +1,71 @@
+# Forms
+
+## Checklist
+
+### Buttons rather than inputs
+
+- Specify the required attributes (name and value)
+
+### Multistep forms over multiple submit buttons
+
+### Server-side Validation
+
+- Start with server-side validation
+- Return an error message for each erroring form element.
+
+### Client-side validation
+
+### Accessibility
+
+### Label every input
+
+### Use the most simple form control that does the job
+
+### Minimise the number of fields to fill out
+
+### Mark optional fields
```

Comment:
- Yes ☀️ it is preferred over the * (required) pattern
- If they are optional, it might be worse to re-evaluate if they are really needed after all and if they might not be getting in the way to conversion